source/gloperate/include/gloperate/rendering/ScreenAlignedQuad.h

```diff
@@ -0,0 +1,70 @@
+
+#pragma once
+
+#include <globjects/Buffer.h>
+#include <globjects/VertexArray.h>
+#include <globjects/base/Referenced.h>
+#include <globjects/base/ref_ptr.h>
+
+#include <gloperate/rendering/AbstractDrawable.h>
+#include <gloperate/rendering/Drawable.h>
+
+#include <gloperate/gloperate_api.h>
+
+
+namespace gloperate
+{
+
+
+/**
+*  @brief
+*    Screen aligned quad rendering helper
+*
+*    This class can be used to render a screen aligned quad.
+*/
+class GLOPERATE_API ScreenAlignedQuad : public globjects::Referenced, gloperate::AbstractDrawable
```

Comment:
The class is defined and declared in namespace gloperate, so there's no need to repeat the namespace (here and on more occasions).